sources/packages/backend/apps/workers/src/controllers/application/_tests_/e2e/application.controller-updateStatus.e2e-spec.ts

```diff
@@ -0,0 +1,133 @@
+import { ApplicationStatus } from "@sims/sims-db";
+import {
+  createE2EDataSources,
+  E2EDataSources,
+  saveFakeApplication,
+} from "@sims/test-utils";
+import { APPLICATION_STATUS_NOT_UPDATED } from "../../../../constants";
+import { IOutputVariables } from "zeebe-node";
+import {
+  createFakeWorkerJob,
+  FAKE_WORKER_JOB_ERROR_CODE_PROPERTY,
+  FAKE_WORKER_JOB_ERROR_MESSAGE_PROPERTY,
+  FAKE_WORKER_JOB_RESULT_PROPERTY,
+  MockedZeebeJobResult,
+} from "../../../../../test/utils/worker-job-mock";
+import { createTestingAppModule } from "../../../../testHelpers";
+import { ApplicationController } from "../../application.controller";
+import {
+  ApplicationUpdateStatusJobHeaderDTO,
+  ApplicationUpdateStatusJobInDTO,
+} from "../../application.dto";
+import { createFakeUpdateApplicationStatusPayload } from "./update-application-status";
+
+describe("ApplicationController(e2e)-updateStatus", () => {
+  let db: E2EDataSources;
+  let applicationController: ApplicationController;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    await nestApplication.init();
```

Comment:
I believe that the `nestApplication.init()` is already happening in `createTestingAppModule`.